### PR TITLE
Makman2/linterext

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

File: coalib/bearlib/abstractions/Linter.py [hunk 1]
```diff
@@ -672,9 +688,12 @@ def linter(executable: str,
...
         defined inside ``coalib.results.RESULT_SEVERITY``.
-    :param diff_message:
-        The message-string to use for all results if ``output_format`` is
-        ``'corrected'``. By default this value is ``"Inconsistency found."``.
+    :param result_message:
+        The message-string to use for all results. Can be used only together
+        with ``corrected`` or ``regex`` output format. When using
+        ``corrected``, the default value is ``"Inconsistency found."``, while
+        for ``regex`` this static message is disabled and the matched message
+        is used instead.
```
Comment (Makman2) [inline coalib/bearlib/abstractions/Linter.py]: maybe I should include "... the message *matched by `output_regex`* is used instead"

**Comment by mr-karan** [comment]
ack https://github.com/coala-analyzer/coala/pull/2159/commits/88a9ca0ff299e5f7eab515ecd606c6370ca339cf

**Comment by mr-karan** [comment]
@rultor merge

**Comment by sils1297** [comment]
@mr-karan you didn't ack the first commit?

**Pull request merged**